sources/sqlserver/README.md

```diff
@@ -70,3 +70,86 @@ for SQLserver source. Password can be provided at the password prompt.
 | HIERARCHYID            | STRING       |
 | Spatial Geography Type | STRING       |
 | Spatial Geometry Types | STRING       |
+
+### `Spatial datatypes`
+
+SQL Server supports `SPATIAL GEOGRAPHY` and `SPATIAL GEOMETRY` datatypes however, Spanner 
+does not support spatial data types.
+These datatype are currently mapped to standard `STRING` Spanner datatype.
+
+### `TIMESTAMP`
+The `TIMESTAMP` datatype (deprecated in the newer versions of SQL Server) 
+was used for Row versioning. Hence, it is mapped to INT64 to keep it consistent
+with the `ROWVERSION` data type.
+
+### Storage Use
+
+The tool maps several SQL Server types to Spanner types that use more storage.
+For example, `SMALLINT` is a two-byte integer, but it maps to Spanner's `INT64`,
+an eight-byte integer.
+
+### Primary Keys
+
+Spanner requires primary keys for all tables. SQL Server recommends the use of
+primary keys for all tables, but does not enforce this. When converting a table
+without a primary key, HarbourBridge will create a new primary key of type
+INT64. By default, the name of the new column is `synth_id`. If there is already
+a column with that name, then a variation is used to avoid collisions.
+Alternatively, when converting a table without a Primary Key having a `UNIQUE`
+constraint, HarbourBridge will automatically choose that column as a Primary Key
+column instead of generating a new one. In case of multiple unique constraints,
```

Comment:
Rephrase - It will pick any unique constraint and convert it to a primary key.

Suggest changing this text into points - so it looks like a step by step process of checks